modules/gltf/structures/gltf_accessor.h

```diff
@@ -31,10 +31,10 @@
 #ifndef GLTF_ACCESSOR_H
 #define GLTF_ACCESSOR_H
 
-#include "../gltf_defines.h"
-
 #include "core/io/resource.h"
 
+#include "../gltf_defines.h"
+
```

Comment:
This should be changed back, the convention for includes in modules is:
- Module-scoped includes first
- Rest of the engine (core, scene, etc.)
- Thirdparty